capellambse/model/_pods.py

```diff
@@ -101,7 +103,9 @@ def __set_name__(self, owner: type[t.Any], name: str) -> None:
         self.__objclass__ = owner
 
     @abc.abstractmethod
-    def _from_xml(self, value: str, /) -> U: ...
+    def _from_xml(
+        self, value: str, model: capellambse.MelodyModel, /
+    ) -> U: ...
```

Comment:
Minor thing: Please change the order to be `self, model, value` instead of `self, value, model`, for more consistency overall. <sub>(Almost)</sub> All functions that require a model expect it as first non-self argument.

It's an API break anyways, as old methods only expect a single argument.